consoles/sshVirtsh.pm

```diff
@@ -322,6 +322,10 @@ sub add_disk {
                 $self->run_cmd("qemu-img create $file $size -f qcow2") && die "qemu-img create failed";
             }
         }
+        elsif ($args->{copy}) {
+            $file = "/var/lib/libvirt/images/$file";
+            $self->run_cmd("rsync -avL $args->{file} $file") && die "rsync failed";
```

Comment:
Only a minor nit, but you could use single quotes for `rsync failed` because no substitution is intended.